lib/src/exception/exception.test.ts

```diff
@@ -52,26 +52,73 @@ describe('SassException', () => {
     }
   });
 
-  it('has the default stack trace', () => {
-    const message = 'default';
-    const error = new SassException(message);
+  describe('stack trace', () => {
+    it('correctly formats the message, span, and trace', () => {
+      const message = 'sad';
+      const sass = 'wrong';
```

Comment:
It's nice to try to re-use these values between the constructor and the expectation, but I think the indirection adds more confusion than it solves (especially since the messages are short and the formatting of the error depends heavily on the structure of the Sass code).

Reply:
Makes sense.